pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>1.1.0</Version>
+    <Version>1.1.1</Version>
```

Comment:
In this diff, we were producing the packages and can you clarify? We will produce packages with `1.1.1-servicing-4158`. When we service, we will turn off all other ones and produce the ones we want. So I don't think we need to pre-emptively do anything here.